src/utils/constant.ts

```diff
@@ -1 +1 @@
-export const Constant = <T>(c:T) => () => c;
\ No newline at end of file
+export const Constant = <T>(c:T) => (...ignored:any) => c;
```

Comment:
wondering the purpose of this addition!

Reply:
It’s used to generate a provider of a constant value do u think it’s silly? ☺️

Reply:
And it works for any number or args so it’s like, when passing a function as a higher order argument to some other function, where u just want it to return a constant value without caring the parameters, it may be clearer for new people not sure may be more confusing. Definitely it’s use is optional!

Reply:
And it works for any number or args so it’s like, when passing a function as a higher order argument to some other function, where u just want it to return a constant value without caring the parameters, it may be clearer for new people not sure may be more confusing. Definitely it’s use is optional!

Reply:
I think the function itself is ok! (though I would understand if someone else said it's overkill) I was more specifically asking about the `...ignored:any` part. I think having an empty `()` is clear enough!

Reply:
I believe I also had a compile error there too. The call site(s) May pass one or two Params? Not sure

Reply:
no compile errors when i remove the stuff inside `()`~

Reply:
killed it!